docs/getting-started/installation.md

```diff
@@ -61,7 +61,7 @@ Next, let's install the latest version of Gaia. Make sure you `git checkout` the
 correct [released version](https://github.com/cosmos/gaia/releases).
 
 ```bash
-git clone -b <latest-release-tag> git@github.com:cosmos/gaia.git
+git clone -b <latest-release-tag> https://github.com/cosmos/gaia.git
```

Comment:
You must add your SSH key to your GitHub for the command works. It's a common thing

Reply:
i agree that it is common for github users to do this. However I also agree that the docs should be set for users with as little contextual knowledge as possible. If someone knows how to set their SSH key in github, they also know that they can change the instructions to `git@github.com...` for the SSH clone command. I think it's a good change to make here and also includes an update to the latest release of v7 family of gaia.